fmc/README.md

```diff
@@ -128,7 +128,8 @@ fields may not be changed or removed). Table revisions with different Major Vers
 | rt_dice_sign          | 96           | FMC        | RT Alias DICE signature.                                                                                 |
 | ldevid_cert_sig_r_dv_hdl | 4         | ROM        | Handle of LDevId Certificate Signature R Component in the Data Vault.                                    |
 | ldevid_cert_sig_s_dv_hdl | 4         | ROM        | Handle of LDevId Certificate Signature S Component in the Data Vault.                                    |
-| idev_dice_pub_key     | 96           | ROM        | Initial Device ID Public Key.                                                                            |
+| idev_dice_ecdsa_pub_key     | 96           | ROM        | Initial Device ID ECDSA Public Key.                                                                            |
+| idev_dice_mlddsa_pub_key_addr     | 4        | ROM        | Address of Initial Device ID MLDSA Public Key.                                                                            |
```

Comment:
I'm not sure how necessary it is to pass the address in the FHT if we're going to use the PersistentData driver.

I think that driver really needs to be kept in backwards compatible with all same-version ROMs anyway (e.g. 2.2 RT cannot overwrite some region that was used by 2.0 ROM).

Reply:
Let's keep this consistent with how other info is shared (e.g. TBS). We can discuss removing as a separate change.